include/dlaf/eigensolver/reduction_to_band/impl.h

```diff
@@ -675,11 +675,12 @@ void hemmComputeX(comm::IndexT_MPI reducer_col, matrix::Panel<Coord::Col, T, D>&
       // directly in x (without using xt)
       const auto i = dist.template localTileFromGlobalTile<Coord::Row>(index_k);
       ex::start_detached(
-          comm::scheduleReduceRecvInPlace(mpi_col_chain(), MPI_SUM, x.readwrite_sender({i, 0})));
+          comm::scheduleReduceRecvInPlace(mpi_col_chain(), MPI_SUM,
+                                          ex::make_unique_any_sender(x.readwrite_sender({i, 0}))));
     }
     else {
-      ex::start_detached(
-          comm::scheduleReduceSend(mpi_col_chain(), rank_owner_row, MPI_SUM, xt.read_sender(index_xt)));
+      ex::start_detached(comm::scheduleReduceSend(mpi_col_chain(), rank_owner_row, MPI_SUM,
```

Comment:
here too...

```suggestion
      ex::start_detached(comm::scheduleReduceSend(ex::make_unique_any_sender(mpi_col_chain()), rank_owner_row, MPI_SUM,
```

Reply:
Closing this as the discussion is the same as in https://github.com/eth-cscs/DLA-Future/pull/674#discussion_r1044555634.